awscli/customizations/wizard/wizards/configure/main.yml

```diff
@@ -0,0 +1,361 @@
+      existing_profiles:
+        type: sharedconfig
+        operation: ListProfiles
+        condition:
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      existing_roles:
+        type: apicall
+        operation: iam.ListRoles
+        params: {}
+        query: "sort_by(Roles[].{display: RoleName, actual_value: Arn}, &display)"
+        cache: true
+        paginate: true
+      credential_source_type:
+        type: prompt
+        description: Select your credentials source type
+        choices:
+          - display: Source Profile
+            actual_value: source_profile
+          - display: Environment Variables
+            actual_value: Environment
+          - display: Amazon EC2 Instance Metadata
+            actual_value: Ec2InstanceMetadata
+          - display: Amazon ECS Container Credentials
+            actual_value: EcsContainer
+        condition:
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      source_profile:
+        type: prompt
+        description: Select the source profile
+        choices: existing_profiles
+        condition:
+          - variable: credential_source_type
+            equals: source_profile
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      role_arn:
+        type: prompt
+        description: Select the role you want to assume
+        choices: existing_roles
+        condition:
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      creds_process:
+        type: prompt
+        description: Enter the credential process command
+        condition:
+          - variable: config_type
+            equals: proc_provider
+          - variable: add_creds
+            equals: yes
+  additional_config:
+    shortname: Additional config
+    description: Additional CLI configuration
+    values:
+      add_config:
+        type: prompt
+        description: "Would you like to add additional configuration"
+        choices:
+          - display: Yes
+            actual_value: yes
+          - display: No
+            actual_value: no
+      region_details:
+        type: template
+        value: |
+          Specifies the AWS Region to send requests to for commands requested using this profile.
+
+            You can specify any of the Region codes available for the chosen service as listed in AWS Regions and Endpoints in the Amazon Web Services General Reference.
+
+            "aws_global" enables you to specify the global endpoint for services that support a global endpoint in addition to regional endpoints, such as AWS Security Token Service (AWS STS) and Amazon Simple Storage Service (Amazon S3).
+      region:
+        type: prompt
+        description: "Enter the region name"
```

Comment:
It would be interesting if we used a `completer` here so that we can get region completions similar to auto-prompt mode. I think that would definitely be helpful.

Reply:
I thought about it but it's not clear what profile to use for the completer client